lib/api/controllers/routerController.js

```diff
@@ -390,6 +408,17 @@ module.exports = function RouterController (kuzzle) {
         _routes[route.controller][route.verb].push('/api/' + kuzzle.config.apiVersion + route.url);
       });
 
+      kuzzle.pluginsManager.routes.forEach(route => {
```

Comment:
To avoid code duplication, this code should be factorized with piece of code just above, probably with a method taking 2 arguments:
* routes (either _routes_ or _kuzzle.pluginsManager.routes_)
* prefix (empty string in the 1st case, _"/_plugin"_ in the second).